flagd/pkg/service/flag-evaluation/ofrep/handler.go

```diff
@@ -57,9 +65,10 @@ func (h *handler) HandleFlagEvaluation(w http.ResponseWriter, r *http.Request) {
 		h.writeJSONToResponse(http.StatusBadRequest, ofrep.ContextErrorResponseFrom(flagKey), w)
 		return
 	}
-
 	context := flagdContext(h.Logger, requestID, request, h.contextValues)
-	evaluation := h.evaluator.ResolveAsAnyValue(r.Context(), requestID, flagKey, context)
+	evaluation := h.evaluator.ResolveAsAnyValue(rCtx, requestID, flagKey, context)
+	span.SetAttributes(attribute.String("feature_flag.key", evaluation.FlagKey))
```

Comment:
i am not sure if we want to add this information here ;)

Reply:
Hmm, I'm not fully up to speed on feature flag semantic conventions, but they [seem to apply to logs rather than spans](https://github.com/open-telemetry/semantic-conventions/blob/6533b8a39e03e6925e080d5ca39234035cf87e70/docs/feature-flags/feature-flags-logs.md). 

Could it be that a log that can be correlated to the `traceId` is what you're trying to write to instead?

Reply:
i will remove those spans for now, and the attributes, i think it is better to have some kind of information, where the span ended up, but nothing more for the begining